packages/bpk-component-calendar-scrollable/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "bpk-component-calendar-scrollable",
```

Comment:
Whilst this component is still unfinished, we should add `"private": true,` here.